Style header and main sections for the first part of the Portfolio Project

xandro2021 opened the pull request:
# Style header and main sections for the first part of the Portfolio Project
in this PR:

+ Set the HTML and CSS Linters
+ Add Normalize stylesheet from https://necolas.github.io/normalize.css/
+ Add some basic global styles to make 1rem equivalent to 10px
+ Create the basic structure of the HTML for the header and the introductory section in the main
+ Rename the feature branch for a proper name that includes both the main section and the header
+ Add some styles at the min break point at 758px for tablets and desktops computers

Thank you for review my PR

xandro2021 committed:
Normalize the default browser styles

+ It was created a exclusive css folder to contain the css files
+ It was downloaded the normalize.css file to delete the default browser
  styles before starting my own personal styles
+ It was include some general styles from an udemy course to make easier
  in styles.css to work with rems
+ It was modified the links routes for the css files in the index.html

xandro2021 committed:
style header

+ Style the header according with the number of px specified by the
  template on figma.
+ Use flex box to position the two icons

xandro2021 committed:
Create html structure in main for the introductory section

+ It was created a section containing a title, to paragraphs and a
  footer

xandro2021 committed:
Rename the branch

+ I'm allow to do only one commit per Review, so I cannot make several
  branche for every feature; therefore, I renamed the branch for a name
  that cover both features

xandro2021 committed:
Set background images for introductory section

+ It was deleted the unnecesary microverse image.
+ It was added two versions of the background for the instroductory
  section. One for small mobil devices and one for desktop and larger
  images.
+ It was modified the index.html to create a better structure for the
  required background

xandro2021 committed:
Add styles at 768px for tablet and desktop users

+ It was added some styles to match the title in the instroductory
section with the figma design, by using a flex wrap with flex basis and
flex direction row instead of just using flex direction column

xandro2021 committed:
Fix linter issues in normalize.css and style.css

+ It was rewrite the troublesome css style rules that were conflicting

Shedrack-Sunday reviewed (changes requested):
Hi @xandro2021 👋 

While you made a great effort in this project, I can't proceed to review your code.

# Invalid Code Review Request 🔴 

You have submitted a pull request that has the README with some missing sections. Kindly provide a detailed description of :
- How to setup the project
- Inclusion of the tech tack section as it is on the README template
- How to use the project under `usage`


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Invalid Code Review Request does not count into the code reviews limit.

xandro2021 committed:
Follow the recommendations of the first code reviewer

Add the following aspects in the README.md
+ Add How to set up the project
+ Include Techstack section again
+ Add information about how to use the project under usage

Kossi-stack commented on index.html:
```diff
@@ -1,14 +1,99 @@
+</head>
+<body>
+    <header class="header container">
+        <div class="logo">
```
- [x] Kindly use a tag for the logo

xandro2021 replied:
@Kossi-stack I don't understand what you mean by tag.
It is to wrap it inside another div tag?

Kossi-stack commented on index.html:
```diff
@@ -1,14 +1,99 @@
+            </svg>
+
+        </div>
+        <div class="icon">
```
- [ ] Kindly wrap the hamburger with a nav tag.

Kossi-stack commented on index.html:
```diff
@@ -1,14 +1,99 @@
+I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+</p>
+
+<div class="icon">
```
- [ ] If there are other titles (e.g., "Let's connect") kindly use a header tag `hx`

Kossi-stack commented on index.html:
```diff
@@ -1,14 +1,99 @@
+
+</div>
+
+<div class="social-media">
```
- [ ] Kindly make sure the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

Kossi-stack reviewed (changes requested):
## Hi @xandro2021,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966](https://user-images.githubusercontent.com/81444338/170121142-a9f93bb7-38bb-45cc-8327-aab2501ef90a.gif)

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly edit the PR's title to something more descriptive. it can be "Portfolio: Setup PR".

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

xandro2021 committed:
Second code review recommendations

+ Wrap the .logo in a tag
+ Wrap the hamburger icon into nav tag
+ Change the svg image for a styled hx
+ Change the div's for ul>li then wrap the icons into a link tag

AbayomiOlaoye commented on index.html:
```diff
@@ -1,99 +1,95 @@
+        <header class="header container">
+            <div>
+                <div class="logo">
+                    <svg width="75" height="18" viewBox="0 0 75 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M14.7121 0.364V13H11.6341V5.422L8.80813 13H6.32413L3.48013 5.404V13H0.402133V0.364H4.03813L7.58413 9.112L11.0941 0.364H14.7121ZM27.1856 2.956L20.8856 17.77H17.5736L19.8776 12.658L15.7916 2.956H19.2296L21.5516 9.238L23.8556 2.956H27.1856ZM35.1911 10.624H39.2231V13H32.1131V0.364H35.1911V10.624ZM45.2632 13.144C44.2792 13.144 43.3912 12.934 42.5992 12.514C41.8192 12.0 ...
+                    </svg>
```
- [ ] I am sorry that you had to go through this stress because of the misconception of what the previous reviewer meant. He was asking you to please use a `a` tag as in an anchor tag (`<a href="index.html">Your Logo Text Here </a>` so that users or visitors can always access your homepage from wherever they are on the page when they click on the logo. :smile: This improves site's accessibility and gives users quality user experience. Now that you have used a `svg` tag in place of texts, kindly wrap the `svg` content in a `a` tag so that users can have blissful experience on your page. By doing this, you will also be implementing one of HTML's best practices :hugs:

xandro2021 replied:
Thank you for the explanation. I finally understand what he wanted.

- [x] wrap the svg content in a a tag so that users can have blissful experience on your page

AbayomiOlaoye reviewed (changes requested):
Hi @xandro2021 :+1: :muscle: , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so_close](https://user-images.githubusercontent.com/99715640/211872359-9a4d36f7-0386-4000-82a4-c274b06a5ce4.gif)


 ## Highlights
- Good PR message ✅ 
- Great commit messages ✅ 
- No linter error ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- You can also consider using images in .png format in order to keep your code neat and tidy. Png images also offer great pixels quality :wink: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

xandro2021 committed:
Wrap the .logo with an ancher tag pointing homepage

bobb-Rob reviewed (approved):
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @xandro2021, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://camo.githubusercontent.com/2c67944c14f5c74b9b607df18d73efb0c423dd148b3e10cb75b445a00220332e/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f5061536e555a573145657970756c734f77642f67697068792e676966)

## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
